libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -753,131 +730,6 @@ string ofSystemTextBoxDialog(string question, string text){
 		 DestroyWindow(dialog);
 		 EnableWindow(WindowFromDC(wglGetCurrentDC()), TRUE);
 
-	#else
-
```

Comment:
oh, i see i saw  such a big chunk of code go away and freaked out :)

yeah i agree on tabs and i don't think there's really a strong preference for them it's just been like that, at some point if we run the styler script on all the code base we can discuss about changing to 4 spaces